### PR TITLE
Add Warnings and Fix Breaks

### PR DESCRIPTION
**Pull request by ruixi1994**
## Proposed changes

Fix Build Break in msal cpp 

## Type of change

- [ ] Feature work
- [X] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [X] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

**Review by jbzdarkid — APPROVED** [review]

**Review by oldalton — APPROVED** [review]

**Comment by kaisong1990** [comment]
@ruixi1994 Could you try to update your branch with the latest dev so that the pipeline will not block you anymore

**Comment by oldalton** [comment]
@ruixi1994, can you merge or close this one? Thanks.

File: IdentityCore/src/telemetry/request_telemetry/MSIDLastRequestTelemetry.m [hunk 1]
```diff
@@ -45,7 +45,6 @@ - (instancetype)initWithCoder:(NSCoder *)decoder
     if (self)
```
Comment (kaisong1990) [inline IdentityCore/src/telemetry/request_telemetry/MSIDLastRequestTelemetry.m:45]: This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.

**Comment by ruixi1994** [comment]
Close as these warnings are addressed with https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/814

**Pull request closed without merging**